andela/Ah-frontend-xmen
# #162725230 Users should follow each other

**What does this PR do?**
This PR enables a logged in user to view profiles and follow/unfollow users.

**Description of tasks completed**
- Create follow, unfollow, following, followers actions
- Create the follow and profileList reducers 
- Add follow/unfollow button
- Add a profile card component
- Return a list of all profiles
- Enable follow and unfollow on multiple profiles

**How this should be manually tested**
- Fetch and check out this branch using `git fetch origin ft-follow-users-162725230 && git checkout ft-follow-users-162725230`
- Log in to Author's Haven. 
- Go to `localhost/who_to_follow` to view a list of all profiles. Click on the `follow` button to follow someone. This changes it to `following`. When `following` is clicked, it changes to `follow` to show unfollowing a user. 
- To see followers of a user, go to `localhost/profiles/username/followers` and the list of following profiles at `localhost/profiles/username/following`, where username is the name of any user. 

**Related Pivotal Tracker Story**
 [#162725230](https://www.pivotaltracker.com/story/show/162725230)
